### PR TITLE
Updated text and logic for judge dashboard and certificate tabs depending on season state

### PR DESCRIPTION
**Pull request by viviancan**
Refs #3939 

A few changes here:
1. Updated judge certificates specs to match current judge experience 
2. The judge text changes were small but there was an opportunity to clean up some of the judge logic. Originally this issue showed up because of a mistake with previous logic that did not display certificates unless judging was on. This has also been fixed and included in this PR. 

CleanShot 2023-04-24 at 16 57 58@2x

CleanShot 2023-04-24 at 16 57 28@2x

CleanShot 2023-04-24 at 16 57 05@2x

### CODE REVIEW

File: app/views/judge/dashboards/onboarded/rebrand/_judging_finished_certs_on.html.erb [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<% if SeasonToggles.display_scores? %>
```
Comment (shaun-technovation) [inline app/views/judge/dashboards/onboarded/rebrand/_judging_finished_certs_on.html.erb:1]: Seems to be an extra conditional here, looks like this conditional is covered in?:
```
    <% if SeasonToggles.display_scores? %>
      <%= render 'judge/dashboards/onboarded/rebrand/judging_finished_certs_on' %>
```

And/or maybe `judge/dashboards/onboarded/rebrand/judging_finished_certs_off.html.erb`should have the extra conditional to be consistent?

File: app/views/judge/dashboards/onboarded/rebrand/_judging_finished_certs_on.html.erb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    </p>
+
+    <ul class="list-disc list-inside mt-4">
+      <li class="pl-4">
+        <%= link_to "Sign up for our newsletter", ENV.fetch("GENERAL_NEWSLETTER_URL"),
+                    class: "tw-link",
+                    target: "_blank" %>
+        to learn more about the girls you helped this season
+      </li>
+      <li class="pl-4"><a class="tw-link" href="https://technovationchallenge.org/get-started/" target="_blank">Consider mentoring</a> a team in the 2023-2024 season. Registration will open in October.</li>
```
Comment (shaun-technovation) [inline app/views/judge/dashboards/onboarded/rebrand/_judging_finished_certs_on.html.erb:26]: The hard-coded `2023-2024` text is popping out to me. 😃

File: app/views/judge/dashboards/onboarded/_dashboard.en.html.erb [hunk 3]
```diff
@@ -12,6 +12,10 @@
   <% elsif SeasonToggles.semifinals_judging? %>
     <%= render 'judge/dashboards/onboarded/rebrand/quarterfinals_or_semifinals' %>
   <% elsif SeasonToggles.judging_finished? %>
-    <%= render 'judge/dashboards/onboarded/rebrand/judging_finished' %>
+    <% if SeasonToggles.display_scores? %>
+      <%= render 'judge/dashboards/onboarded/rebrand/judging_finished_certs_on' %>
```
Comment (shaun-technovation) [inline app/views/judge/dashboards/onboarded/_dashboard.en.html.erb:16]: The `certs_on` and `certs_off` naming here is throwing me off, I think it's just a side effect of how `display_scores?` controls both scores and certificates, but I think it's a little confusing how it reads "display_scores" -> "certs_on". One suggestion, off the top of my head is to create an alias for `display_scores?` and name it `display_certs?`, or maybe we could look at renaming/aliasing `display_scores?` to `display_scores_and_certs?`. Anywho, just a random comment, not a blocker!

**Review by shaun-technovation — APPROVED** [review]
I left few random/non-blocking comments, but over all LGTM! 🙌 👍 🎉 👏

File: app/views/judge/dashboards/onboarded/_dashboard.en.html.erb (hunk 3 above)
Comment (viviancan) [reply app/views/judge/dashboards/onboarded/_dashboard.en.html.erb:16]: 👍 Good idea! I added an alias `display_scores_and_certs?` for `display_scores?`

File: app/views/judge/dashboards/onboarded/rebrand/_judging_finished_certs_on.html.erb (hunk 2 above)
Comment (viviancan) [reply app/views/judge/dashboards/onboarded/rebrand/_judging_finished_certs_on.html.erb:26]: Good suggestion! Updated this!

File: app/views/judge/dashboards/onboarded/rebrand/_judging_finished_certs_on.html.erb (hunk 1 above)
Comment (viviancan) [reply app/views/judge/dashboards/onboarded/rebrand/_judging_finished_certs_on.html.erb:1]: Totally missed that! Thanks, Shaun!

**Pull request merged**